torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -0,0 +1,162 @@
+from typing import Optional
+
+from torch.nn.modules.batchnorm import BatchNorm2d
+from torch.nn.modules.instancenorm import InstanceNorm2d
+from torchvision.models.optical_flow import RAFT, BottleneckBlock, ResidualBlock
+from torchvision.models.optical_flow.raft import _raft
+from torchvision.prototype.transforms import RaftEval
+
+from .._api import WeightsEnum, Weights
+
+
+__all__ = (
+    "RAFT",
+    "raft_large",
+    "raft_small",
+)
+
+
+class Raft_Large_Weights(WeightsEnum):
+    C_T = Weights(
+        # Chairs + Things
+        url="",
+        transforms=RaftEval,
+        meta={
+            "recipe": "",
+            "epe": -1234,
+        },
```

Comment:
I'll fill these up ASAP when I get the results and when https://github.com/pytorch/vision/pull/5027 and https://github.com/pytorch/vision/pull/5026 are merged.

@datumbox can the `meta` field support epe for one than one dataset?
I assume I can just have different keys like "sintel_train_epe", "sintel_test_epe", "kitti_train_epe", etc?